packages/integrations/mdx/README.md

```diff
@@ -222,6 +223,46 @@ const { frontmatter } = Astro.props;
 </html>
 ```
 
+You can also add type safety using [the `Props` type](/en/guides/typescript/#component-props) with the `MDXLayoutProps` helper.
+
+_**Note:** `MDXLayoutProps` is the same as the `MarkdownLayoutProps` utility type, with `rawContent()` and `compiledContent()` removed (since these are not available for `.mdx` files). If you understand this difference, feel free to use `MarkdownLayoutProps` instead when sharing a layout across `.md` and `.mdx` files._
+
+```astro ins={2,4-9}
+---
+// src/layouts/BaseLayout.astro
+import type { MDXLayoutProps } from 'astro';
+
+type Props = MDXLayoutProps<{
+  // Define frontmatter props here
+  title: string;
+  author: string;
+  date: string;
+}>;
+
+// Now, `frontmatter`, `url`, and other MDX layout properties
+// are accessible with type safety
+const { frontmatter, url } = Astro.props;
+---
+<html>
+  <head>
+    <meta rel="canonical" href={new URL(url, Astro.site).pathname}>
+    <title>{frontmatter.title}</title>
+  </head>
+  <body>
+    <h1>{frontmatter.title}</h1>
+    <slot />
+  </body>
+</html>
+```
+
+#### Layout props
+
+All [exported properties](#exported-properties) are available from `Astro.props` in your layout, **with a couple key differences:**
```

Comment:
```suggestion
All [exported properties](#exported-properties) are available from `Astro.props` in your layout, **with two key differences:**
```
If I'm feeling generous, I would also accept "a couple of." But, if it doesn't offend your sensibilities, let's say "two."

Reply:
Ah man, can't even get generous Sarah on a Friday? Fine 🙄

Reply:
(You *said* fine.... but I don't see this changed....)  ;)